data-rights-protocol.md

```diff
@@ -291,7 +348,7 @@ Covered Businesses SHALL determine for themselves the level of reliance they wil
 
 Servers SHALL respond with HTTP 200 response codes when requests are processed successfully. In exceptional cases, servers SHALL respond with non-200 response codes and an `application/json` body with the following keys:
 
-- `code` MUST contain a string encoding of the HTTP response code for clients which cannot process the headers. 
+- `code` MUST contain a string encoding of the HTTP response code for clients which cannot process the headers.
```

Comment:
What is the intention behind this? I can't think of any client which can't access this information, and this seems pretty open to confusion to me, e.g. which should a client trust if the status code is 200 but the `code` key is 404 or vice versa?